src/main/resources/db/local/V99_2__draft_referrals.sql

```diff
@@ -0,0 +1,23 @@
+insert into referral (id, intervention_id, created_at, sent_at, service_usercrn, completion_deadline, created_by_id, further_information, additional_needs_information, accessibility_needs, needs_interpreter, interpreter_language, has_additional_responsibilities, when_unavailable, additional_risk_information, using_rar_days, maximum_rar_days, complexity_levelid, reference_number)
+values ('ac386c25-52c8-41fa-9213-fcf42e24b0b5', '98a42c61-c30f-4beb-8062-04033c376e2d', '2020-12-07 18:02:01.599803+00', null, 'X987623', '2021-02-14', '2500128586', null, null, null, null, null, null, null, null, null, null, null, null),
```

Comment:
_not a blocker_
Hmm, perhaps these would be easier to create in code now? As fixtures/sample data/seeds?

This is reaching 'too much detail to follow' level for me.